.dockerignore

```diff
@@ -0,0 +1,8 @@
+*.pyc
+.ipynb_checkpoints
+.python-version
+__pycache__/
+config.ini
```

Comment:
Doesn't really matter, the current dir will be mounted inside the container and people are supposed to `cp config.ini.example config.ini` anyways ;)